python/cudf/cudf/core/dataframe.py

```diff
@@ -695,6 +695,7 @@ def __init__(
         index=None,
         columns=None,
         dtype=None,
+        copy=None,
```

Comment:
Is `copy` meant to be non-functional for now? Should we raise NotImplementedError ?

Reply:
Yeah good catch; this is currently not implemented. I'll make it raise a NotImplementedError